perception/traffic_light_occlusion_predictor/src/occlusion_predictor.cpp

```diff
@@ -107,7 +111,7 @@ void CloudOcclusionPredictor::predict(
     lidar_rays_[static_cast<int>(ray.azimuth)][static_cast<int>(ray.elevation)].push_back(ray);
   }
   for (size_t i = 0; i < roi_tls.size(); i++) {
-    occlusion_ratios[i] = predict(roi_tls[i], roi_brs[i]);
+    occlusion_ratios[i] = rois_msg->rois[i].roi.height == 0 ? 1.0 : predict(roi_tls[i], roi_brs[i]);
```

Comment:
[Q]
The occlusion_ratio is an integer, and 1 represents 1%. Is this the intended value?

Reply:
I'm afraid not. It should be 0 otherwise the unknown signal will be set as unknown again.

Reply:
fixed inhttps://github.com/autowarefoundation/autoware.universe/pull/5934/commits/d96975bfa8b7bd6c158e84141815891023c2591c